src/main/java/teammates/ui/webapi/InvalidOperationException.java

```diff
@@ -16,7 +16,7 @@ public InvalidOperationException(String message) {
         super(message);
     }
 
-    public InvalidOperationException(EntityAlreadyExistsException cause) {
+    public <T extends Exception> InvalidOperationException(T cause) {
```

Comment:
This means that `InvalidOperationException` can wrap any Exception; that's not an acceptable design.